pkg/module/remote.go

```diff
@@ -100,9 +100,7 @@ func (r *Remote) getCredentials(ctx cpi.Context) credentials.Credentials {
 	return creds
 }
 
-// See: github.com/open-component-model/ocm/pkg/contexts/credentials/repositories/dockerconfig/repository.go#IsEmptyAuthConfig()
```

Comment:
Maybe leave this  comment? Tt indicates that the function is written as the one linked.

Reply:
The link does not work, that's why I removed it.